### DISCUSSION TITLE
Poll: Do you need a tutorial for ZAP?

### DISCUSSION THREAD

**Discussion by Boganabi** [DrAlzahraniProjects/csusb_fall2024_cse6550_team1#35] [Polls]
Poll: Do you need a tutorial for ZAP?

**Comment by Sai0235** [comment]
Yes, Logan I need a tutorial for ZAP

**Comment by Sai0235** [comment]
When we have to use the zap in our project by the end? we have to test every files, zap is not getting installed in my laptop,help me.
